hedera-node/hapi-fees/src/main/java/org/hiero/hapi/fees/FeeModel.java

```diff
@@ -14,11 +14,11 @@
  */
 public interface FeeModel {
     /** Get the @{@link HederaFunctionality} that this fee module handles. */
-    HederaFunctionality getApi();
+    String getApi();
 
     /** A string description of this fee model. Used for human-readable output. */
     String getDescription();
 
     /** Compute the fee for a specific transaction. */
-    FeeResult computeFee(Map<Extra, Object> params, FeeSchedule feeSchedule);
+    FeeResult computeFee(Map<Extra, Long> params, FeeSchedule feeSchedule);
```

Comment:
Why do we need to pass `FeeSchedule feeSchedule` here? Isn't this known already. We can inject that FeeSchedule through Dagger into StandardFeeModel?